File: aoc2017/day4/part1.py
def main():
    with open('input.txt','r') as f:
        p = [line.strip() for line in f]
    
    print(p)
    count = 0
    for password in p:
        items = password.split(" ")
        l1 = len(set(items))
        l2 = len(items)
        if(l1==l2):
            count+=1

    print(count)

main()

File: aoc2022/day1/part1.js
import fs from "fs";

const calorieSum = (nums) => nums.reduce((a, b) => a + b, 0);

const solve = (fileName) => {
  const fileContent = fs.readFileSync(fileName, "utf-8");
  const numbers = fileContent
    .split("\n\n")
    .map((line) => line.split("\n").map(Number))
    .map(calorieSum);
  return numbers;
};

console.log(Math.max(...solve("./input.txt")));
